src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -164,16 +160,15 @@ public function editAction(Post $post, Request $request)
 
         return $this->render('admin/blog/edit.html.twig', [
             'post' => $post,
-            'edit_form' => $editForm->createView(),
-            'delete_form' => $deleteForm->createView(),
+            'form' => $form->createView(),
         ]);
     }
 
     /**
      * Deletes a Post entity.
      *
      * @Route("/{id}", name="admin_post_delete")
-     * @Method("DELETE")
+     * @Method("POST")
```

Comment:
Why don't you like the `DELETE` method? Is because browsers don't support it and we must simulate it with the `_method` trick? Thanks!